webots_ros2_driver/webots_ros2_driver/webots_launcher.py

```diff
@@ -82,3 +95,70 @@ def __init__(self, output='screen', world=None, gui=True, mode='realtime', strea
             ],
             **kwargs
         )
+
+    def execute(self, context: LaunchContext):
+        # User can give a PathJoinSubstitution world ...
+        if isinstance(self.__world, PathJoinSubstitution):
+            world_path = self.__world.perform(context)
+            context.launch_configurations['world'] = self.__world_copy.name
+        # or an absolute path world
```

Comment:
I don't understand this comment? Is it on the right line? What does it mean?

Reply:
It goes with line 100, but I will put these 2 comments on 1 line to make it clearer.